projects/1.19/assets/dense-ores-refabricated/denseores/lang/zh_cn.json

```
@@ -0,0 +1,109 @@
+{
+  "block.denseores.dense_coal_ore": "致密煤矿石",
+  "block.denseores.dense_copper_ore": "致密铜矿石",
+  "block.denseores.dense_iron_ore": "致密铁矿石",
+  "block.denseores.dense_gold_ore": "致密金矿石",
+  "block.denseores.dense_redstone_ore": "致密红石矿石",
+  "block.denseores.dense_lapis_ore": "致密青金石矿石",
+  "block.denseores.dense_diamond_ore": "致密钻石矿石",
+  "block.denseores.dense_emerald_ore": "致密绿宝石矿石",
+
+  "block.denseores.dense_deepslate_coal_ore": "致密深层煤矿石",
+  "block.denseores.dense_deepslate_copper_ore": "致密深层铜矿石",
+  "block.denseores.dense_deepslate_iron_ore": "致密深层铁矿石",
+  "block.denseores.dense_deepslate_gold_ore": "致密深层金矿石",
+  "block.denseores.dense_deepslate_redstone_ore": "致密深层红石矿石",
+  "block.denseores.dense_deepslate_lapis_ore": "致密深层青金石矿石",
+  "block.denseores.dense_deepslate_diamond_ore": "致密深层钻石矿石",
+  "block.denseores.dense_deepslate_emerald_ore": "致密深层绿宝石矿石",
+
+  "block.denseores.dense_nether_quartz_ore": "致密下界石英矿石",
+  "block.denseores.dense_nether_gold_ore": "致密下界金矿石",
+  "block.denseores.dense_ancient_debris": "致密远古残骸",
+
+
+  "itemGroup.denseores.item_group": "致密矿石：Fabric版",
+
+
+  "configOption.denseores.canGenerate.tooltip": "确定此矿石是否可以在世界中生成",
+  "configOption.denseores.canGenerate.tooltip.2": "此矿石在原版中不会生成，请勿启用，除非你使用了自定义世界生成。",
+  "configOption.denseores.veinSize.tooltip": "决定单个矿脉中可以生成多少数量的这种矿石",
+  "configOption.denseores.minHeight.tooltip": "此矿石能够生成的最低Y坐标",
+  "configOption.denseores.maxHeight.tooltip": "此矿石能够生成的最高Y坐标",
+  "configOption.denseores.chunkOdds.tooltip": "此矿石在一个区块内生成的概率。数值越高，生成概率越大",
+
+
+  "configOption.denseores.canGenerateDenseCoal": "生成致密煤矿石？",
+  "configOption.denseores.coalVeinSize": "致密煤矿石的矿脉尺寸",
+  "configOption.denseores.coalMinHeight": "致密煤矿石的最低高度",
+  "configOption.denseores.coalMaxHeight": "致密煤矿石的最大高度",

```

```suggestion
  "configOption.denseores.coalMaxHeight": "致密煤矿石的最高高度",
```
属于是强行对齐了（）
如果更改则全文通查